README.md

```diff
@@ -0,0 +1,54 @@
+# Admission trial project
+
+> Set up a Project2-trial folder.
+
+![image](https://user-images.githubusercontent.com/66781780/124811245-4eba5700-df73-11eb-8a75-484d4cce4dcc.png)
+
+In this project, we will set up a "A copy-admission-trial-project" repository. No complex coding is required for this exercise. Our goal here is to master all of the tools and best practices we learned about in previous steps.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+`https://github.com/balqees0325/project2-trial.git`
+
+change into the working directory.
+
+`cd project2-trial`
+
+### Prerequisites
+
+- Node.JS
+
+## Author
+
+👤 **Balqees Ibrahim**
+
+- GitHub: [@balqees0325](https://github.com/balqees0325)
+- Twitter: [@qeescode](https://twitter.com/qeescode)
+- LinkedIn: [Balqees Ibrahim](https://www.linkedin.com/in/balqees-ibrahim-911a81204/)
+
+## Live Demo
```

Comment:
Well done for a good job.
However the live link on the readme file is not working, try to check that.